components/ArticleReplyFeedbackControl/ReasonsDisplay.js

```diff
@@ -95,9 +95,21 @@ function ReasonsDisplay({ articleReply, onSizeChange = () => {} }) {
     if (onSizeChange) return onSizeChange();
   }, [tab, onSizeChange]);
 
+  const IsEmptyComment = comment => comment === '' || comment === null;
+
   const feedbacks =
     data?.ListArticleReplyFeedbacks?.edges.map(({ node }) => node) || [];
 
+  const processedFeedbacks = (feedbacks, voteType, isLoadMore) =>
+    feedbacks
+      .filter(({ vote }) => vote === voteType)
+      .sort((a, b) => {
+        if (IsEmptyComment(a.comment)) return 1;
+        else if (IsEmptyComment(b.comment)) return -1;
+        return 0;
```

Comment:
nit: I am thinking if we should sort feedbacks using time (can use [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) on `.createdAt` or `.updatedAt) when both `a` and `b` are populated (or when both are not populated)

Reply:
I've used createdAt as sorted key, the newest feedbacks will go first